Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,5783 @@
+    if version == ClientV1.API_VER:
+        headers = headers + v1_only_headers
+    if version == ClientV2.API_VER:
+        headers = headers + v2_only_headers
+    return headers
+
+
+def paginate_results(version: str, response: Union[List, Dict[str, Any]], args: Dict[str, Any]) -> Tuple[list, str]:
+    """ Executing Manual paginate_results  (using the page and page size arguments)
+        or Automatic paginate_results  (display a number of total results).
+    Args:
+        response (Union[Dict[str, Any], List[Dict[str, Any]]]): API response.
+        page (int, optional): Page number of paginated results.
+        page_size (int, optional): Number of ip-list per page.
+        limit (int, optional): The maximum number of records to retrieve.
+    Returns:
+        Tuple[dict,str]: Output and paginate_results  message for Command Results.
+    """
+    # if version == ClientV2.API_VER:
+    #     response = response['results']  # type: ignore # V2 always returns a Dict.
+    response = response if isinstance(response, list) else [response]
+    page = arg_to_number(args.get('page'))
+    page_size = arg_to_number(args.get('page_size'))
+    limit = arg_to_number(args.get('limit', LIMIT_SIZE))
+
+    output = response
+
+    if page and page_size:
+        if page_size < len(response):
+            first_item = page_size * (page - 1)
+            output = response[first_item:first_item + page_size]
+        else:
+            output = response[:page_size]
+        pagination_message = f'Showing page {page}. \n Current page size: {page_size}'
+    else:
+        output = response[:limit]
+        pagination_message = f'Showing {len(output)} rows out of {len(response)}.'
+
+    return output, pagination_message
+
+
+def find_dict_in_array(container: List[Dict[str, Any]], key: str, value: str) -> Optional[Dict[str, Any]]:
+    """Gets dictionary object in list of dictionaries.The search is by key that exist in each dictionary.
+
+    Args:
+        container (list): List of dictionaries.
+        key (str): Key to recognize the correct dictionary.
+        value (str]): The value for the key.
+
+    Returns:
+        Optional[Dict[str, Any]]: The dictionary / The dictionaries / None if there is no match.
+    """
+
+    for obj in container:
+        if obj.get(key) and str(obj[key]) == value:
+            return obj
+    return None
+
+
+def find_dicts_in_array(container: List[Dict[str, Any]], key: str, value: List[str]) -> Optional[List[Dict[str, Any]]]:
+    """Gets dictionaries object in list of dictionaries.The search is by key that exist in each dictionary.
+
+    Args:
+        container (list): List of dictionaries.
+        key (str): Key to recognize the correct dictionary.
+        value (List[str]): The values for the key.
+
+    Returns:
+        Optional[List[Dict[str, Any]]]: The dictionaries / None if there is no match.
+    """
+
+    return [obj for obj in container if obj[key] in value] or None
```

Comment:
```suggestion
    return [obj for obj in container if obj[key] in value] 
```
I think that this is enough. in case the list is not populated, an if statement: if my_list - will be evaluated as False.